src/super_gradients/training/datasets/datasets_conf.py

```diff
@@ -35,3 +35,8 @@
     'cell phone', 'microwave', 'oven', 'toaster', 'sink', 'refrigerator', 'book', 'clock', 'vase', 'scissors',
     'teddy bear', 'hair drier', 'toothbrush'
 ]
+
+COCO_91_INDEX = [
```

Comment:
Don't think we really need this (the name is pretty confusing as well).
We can just take the class indices from the segmentation classes- COCO_DEFAULT_CLASSES_TUPLES_LIST without the background class, can't we ?)

Reply:
I removed COCO_91_INDEX  from datasets_conf and generated it like that:
`
        coco_detection_category_id_list = [
            category_id
            for category_id, category_name in COCO_DEFAULT_CLASSES_TUPLES_LIST
            if category_name in COCO_DETECTION_CLASSES_LIST
        ]
`

see https://github.com/Deci-AI/super-gradients/blob/d0c79c703fb824c3ec4c6c4f01b241c27cdfaeae/src/super_gradients/training/datasets/detection_datasets/coco_detection.py